File: python/8kyu/even_or_odd.py
"""
Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
"""


def even_or_odd(number: int):
    return 'Odd' if number % 2 else 'Even'


some_number = int(input('Enter some number: '))
print(even_or_odd(some_number))

File: javascript/8kyu/even_or_odd.js
/*
Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
 */

// function even_or_odd(number) {
//     if (number % 2 === 0) {
//         return 'Even'
//     } else {
//         return 'Odd'
//     }
// }

function evenOrOdd(number) {
    return number % 2 ? "Odd" : "Even";
}

console.log(evenOrOdd(2)) // 'Even'
console.log(evenOrOdd(7)) // 'Odd'